03_hardware_configuration/slides/slides.md

```diff
@@ -0,0 +1,631 @@
+  <br>
+  <div class="grid-two-col-eq">
+    <div class="flex-col justify-center">
+    <p>Roles are configured in the hwl.yml file. A <b>"UserManagement"</b> section is created when defining the PLC. There, roles can be defined and then assigned to users.</p>
+    <p>The function roles can come from the following things, for example.:</p>
+    <ul>
+     <li><a href="https://console.simatic-ax.siemens.io/docs/hw/security/authorization/plc-access-protection">The Access to PLC</a></li>
+     <li><a href="https://console.simatic-ax.siemens.io/docs/hw/security/authorization/plc-web-server">PLC Web Server Authorization</a></li>
+     <li><a href="https://console.simatic-ax.siemens.io/docs/hw/security/authorization/opc-ua-server">OPC UA Server Authorization</a></li>
+   </ul>
+    </div>
+    <img style="height: 350px; width: auto" src="./img/UMAC_in_hwl_yml.png" />
+  </div>
+  <ol>
+    <li><br>For More Informations about UMAC on the AX Side: <br><a href="https://console.simatic-ax.siemens.io/docs/hw/security/authorization/user-management#user-management-access-control-umac">User Management Access Control (UMAC)</a></li> 
+  </ol>
+</div>
+
+----
+
+<header class="slide_header">
+    <h3>User Management UMAC - <br>Managing User Credentials over command line of hwc</h3>
+</header>
+
+<div class="flex-col justify-center">
+  <p>Using the command <b>"apax hwc manage-users"</b>, you can list all known users, assign passwords, and create users. As with the other commands, you must specify the PLC name and the location of the PLC config file the user is currently in. Note: Role assignment is still done via the hwl condig file. Therefore, both the config file and user management in the PLC Security Configuration File must be maintained.</p>
+  <br>
+  <p> The command now offers the following options:
+    <ul>
+      <li><b>list:</b> <br>Lists all users for which credentials have been configured for the specified PLC in the PLC Security Configuration File.</li>
+      <li><b>set-password:</b> <br>Sets the password for a PLC's user. If the user does not yet exist, it will be created implicitly. If the user already exists, the previously configured password will be overwritten..</li>
+      <li><b>delete:</b> <br>Deletes a user from the PLC Security Configuration File.</li>
+    </ul>
+  </p>
+  <ol>
+    <li><br>For More Informations about mange User Command on AX Side: <br><a href="https://console.simatic-ax.siemens.io/docs/hw/hwc#manage-users-command">Manage User Command</a></li> 
+  </ol>
+
+</div>
+
+
+---
+
+<header class="slide_header">
+    <h2>IT-Like Hardware Engineering</h2>
+    <h3>Hardware Device Identifiers assignment</h3>
+</header>
+
+<div class="flex-col justify-center">
+  <p>When you set up or assemble a system for the first time, it's important to assign basic identifiers, such as names or IP addresses, to the modules, since the modules are delivered with the same values from the factory. This applies to the spare parts use case. <br>
```

Comment:
```suggestion
  <p>When you set up or assemble a system for the first time, it's important to assign communication addresses, such as the IP address, to the modules, since the modules are delivered with the same values from the factory. This applies to the spare parts use case. <br>
```